core/store/src/trie/iterator.rs

```diff
@@ -10,7 +10,7 @@ struct Crumb {
     status: CrumbStatus,
 }
 
-#[derive(Clone, Eq, PartialEq, Debug)]
+#[derive(Clone, Copy, Eq, PartialEq, Debug)]
```

Comment:
I would like to understand the pros and cons of adding `Copy` here.  My understanding is that deriving `Copy` enables one to implicitly duplicate types.  So for simple types, that can be desirable as it reduces amount of code.  Is that the main benefit?

Reply:
Semantic wise, yes, `Copy` makes the code simpler: no need for explicit `.clone` calls, or references with lifetimes. So, "if it can be copy, make it Copy" is a good first-order guideline. 

Some finer points are: 

* for types with big size_of, there might be some non-trivial perf implications (both ways): https://github.com/near/nearcore/pull/5577#issuecomment-984630948
* if something is `Copy`, *removing* `Copy` usually requires substantial modification of all call-sites. So, for libraries or pervasive types, it sometimes makes sense to not implement Copy for future proofing

Reply:
Also subsequently found a link to https://github.com/near/nearcore/pull/5574.  Interesting discussions.  Given that this does not appear to be pervasive and not part of the public API, I think it should be easy to make changes here if we need to.  I'll approve.